src/main/app/src/app/shared/material-form-builder/form-components/html-editor/html-editor-form-field.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {FieldType} from '../../model/field-type.enum';
+import {ActionIcon} from '../../../edit/action-icon';
+import {AbstractFormControlField} from '../../model/abstract-form-control-field';
+
+export class HtmlEditorFormField extends AbstractFormControlField {
+    fieldType: FieldType = FieldType.HTML_EDITOR;
+    icons: ActionIcon[];
+    maxlength: number;
+    showCount: boolean = false;
```

Comment:
In gevallen waarbij we direct de waarde zetten, zetten  we het type er niet bij. Dus showCount = false;